### DISCUSSION TITLE
How can I reset the use of filters, brushes, etc ...?

### DISCUSSION THREAD

**Discussion by spunchybunchy** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#257] [Q&A] [answered]
Well, how can I reset the use of filters, brushes, etc ...? Yesterday I changed some spare parts of the vacuum cleaner but I do not know how to reset them in the integration.
My model is a: roborock vacuum s5
Thanks a lot.

**Comment by PiotrMachowski** [comment, marked as answer]
https://www.home-assistant.io/integrations/xiaomi_miio/#example-on-how-to-reset-maintenance-hours-brushes-filter-sensors

**Reply by spunchybunchy** [reply]
Ok. Thank you. And thank you por you speed

**Reply by spunchybunchy** [reply]
I have already programmed the script and it works fine. Now I make a request that I think would be interesting for the project: that when holding the button of each replacement it is reset to 0 and so it would be all integrated into the same card.
I do not know if it is possible but I think it would complement the integration.

Thanks a lot.

**Reply by PiotrMachowski** [reply]
@spunchybunchy it's already possible, every tile can have touch/hold/double click action defined, but you will have to override default generated tiles
